packages/types/src/allForks/sszTypes.ts

```diff
@@ -13,20 +14,38 @@ export const allForks = {
     SignedBeaconBlock: phase0.SignedBeaconBlock,
     BeaconState: phase0.BeaconState,
     Metadata: phase0.Metadata,
+    // Not used in phase0 but added for type consitency
+    ExecutionPayload: bellatrix.ExecutionPayload,
+    ExecutionPayloadHeader: bellatrix.ExecutionPayloadHeader,
   },
   altair: {
     BeaconBlockBody: altair.BeaconBlockBody,
     BeaconBlock: altair.BeaconBlock,
     SignedBeaconBlock: altair.SignedBeaconBlock,
     BeaconState: altair.BeaconState,
     Metadata: altair.Metadata,
+    // Not used in altair but added for type consitency
+    ExecutionPayload: bellatrix.ExecutionPayload,
+    ExecutionPayloadHeader: bellatrix.ExecutionPayloadHeader,
```

Comment:
to be dropped after rebase right?

Reply:
i plan  to move them to a separate allTypesExecution as some fork based usage is there